experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/proxies.spec.ts

```diff
@@ -52,64 +89,70 @@ describe("SharedTreeObject", () => {
 		list: [{ content: 42 }, { content: 42 }],
 	};
 
-	function itWithRoot(
-		title: string,
-		fn: (root: ProxyField<(typeof schema)["rootFieldSchema"]>) => void,
-	): void {
-		it(title, () => {
-			const view = createTypedTreeView(schema, initialTree);
-			const root = view.root2(schema);
-			fn(root);
-		});
-	}
-
-	itWithRoot("can read required fields", (root) => {
+	itWithRoot("can read required fields", schema, initialTree, (root) => {
 		assert.equal(root.content, 42);
 		assert.equal(root.child.content, 42);
 	});
 
-	// TODO: Enable when implemented
-	// itWithRoot("can read lists", (root) => {
-	// 	assert.equal(root.list.length, 2);
-	// 	for (const x of root.list) {
-	// 		assert.equal(x, 42);
-	// 	}
-	// });
+	itWithRoot("can read lists", schema, initialTree, (root) => {
+		assert.equal(root.list.length, 2);
+		for (const x of root.list) {
+			assert.equal(x.content, 42);
+		}
+	});
 
-	itWithRoot("can read fields common to all polymorphic types", (root) => {
+	itWithRoot("can read fields common to all polymorphic types", schema, initialTree, (root) => {
 		assert.equal(root.polyChild.content, "42");
 	});
 
-	itWithRoot("can narrow polymorphic value fields", (root) => {
+	itWithRoot("can narrow polymorphic value fields", schema, initialTree, (root) => {
 		if (typeof root.polyValue === "number") {
 			assert.equal(root.polyChild.content, 42);
 		} else {
 			assert.equal(root.polyChild.content, "42");
 		}
 	});
 
-	itWithRoot("can narrow polymorphic struct fields", (root) => {
+	itWithRoot("can narrow polymorphic struct fields", schema, initialTree, (root) => {
 		if (is(root.polyChild, numberChild)) {
 			assert.equal(root.polyChild.content, 42);
 		} else {
 			assert.equal(root.polyChild.content, "42");
 		}
 	});
 
-	itWithRoot("can narrow polymorphic combinations of value and struct fields", (root) => {
-		if (is(root.polyValueChild, numberChild)) {
-			assert.equal(root.polyValueChild.content, 42);
-		} else {
-			assert.equal(root.polyValueChild, 42);
-		}
+	itWithRoot(
+		"can narrow polymorphic combinations of value and struct fields",
+		schema,
+		initialTree,
+		(root) => {
+			if (is(root.polyValueChild, numberChild)) {
+				assert.equal(root.polyValueChild.content, 42);
+			} else {
+				assert.equal(root.polyValueChild, 42);
+			}
+
+			if (typeof root.polyValueChild === "number") {
+				assert.equal(root.polyValueChild, 42);
+			} else {
+				assert.equal(root.polyValueChild.content, 42);
+			}
+		},
+	);
+});
 
-		if (typeof root.polyValueChild === "number") {
-			assert.equal(root.polyValueChild, 42);
-		} else {
-			assert.equal(root.polyValueChild.content, 42);
-		}
+function itWithRoot<TRoot extends FieldSchema>(
+	title: string,
+	schema: TypedSchemaCollection<TRoot>,
+	initialTree: any,
```

Comment:
I was going to give you a hard time about 'any', then saw that I did that too.  :-)

I think 'SchemaAware.TypedField<TRoot, SchemaAware.ApiMode.Flexible>' is still the right typing for the initialTree.